templates/.ohwr.yaml

```diff
@@ -0,0 +1,34 @@
+---
+# SPDX-FileCopyrightText: 2023 CERN (home.cern)
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+# The version of the .ohwr.yaml template.
+version: '1.0.0'
+
+project:
```

Comment:
there should also be a number of "special" links here:

1. home page (not optional I guess, they should at least provide a web view of their repo if they don't have a proper home page)
2. issue/bug tracker (optional)
3. community (optional and if discourse is enabled, then we point to that)
4. download latest release (optional)

Furthermore, there should be a list of licences (1 or more) used in the project.

Reply:
Maybe the home page could fallback to the repo url in case they don't set it?

Is the list of licenses obligatory or optional?

Reply:
For the home page fallback, if it is a git url, then it's not renderable by a browser, so what's the point of putting it?

For the license, I would say yes, the project should state what license it is using.

I have another controversial idea for the licencing; what if we demand that their repo passes reuse lint before we accept to render it and then we use the reuse tool itself to get a list of licences being used? It would be very harsh/strict but effective in keeping everything clean licence-wise. I image that CERN OSPO would also be interested in such an approach ("if you want your project to be published open, you need to take care of the licence").

Reply:
Just added some new fields